### DISCUSSION TITLE
Do I need ModMenu jar for the ModMenu Feature?

### DISCUSSION THREAD

**Discussion by Tim-Odermatt** [Sinytra/ConnectorExtras#27] [Q&A]
I dont quite understand how the Modmenu Integration works.

Does the Integration just add Support to the Fabric mods in order for them to use the ForgeModlistConfig, or can i use ModMenu to replace the ForgeModlist and get all the Forge mods Configs in ModMenu?

When i use Connector+ModMenu without ConnectorExtras, ModMenu Loads correctly(has no Forge Config support).
But when I add ConnectorExtras, ModMenu does not load?

**Comment by coco875** [comment]
ConnectorExtra simulate the precence of mod menu so it's only needed when a mod require modmenu to simulate it without actually have it.

**Reply by coco875** [reply]
ModMenu are not compat with forge because like mentionned forge already have his own mod-menu so it's only here to briges the modmenu api with forgemodmenu
